lenses/connect-customlens-globalResiliency.json

```diff
@@ -26,7 +26,7 @@
 						},
 						{
 							"id": "choice2",
-							"title": "Created a test SAML 2.0-enabled Amazon Connect instance in regions where multi-region is available (us-east-1/ us-west-2)",
+							"title": "Created a test/dev SAML 2.0-enabled Amazon Connect instance in regions where ACGR is available (IAD/PDX or LHR/FRA)",

```

Comment:
We don't define regions like this externally, so we should revert back to terms like us-east-1

Reply:
Updated in the new commit